config/clients/python/template/README_retries.mustache

```diff
@@ -0,0 +1,28 @@
+Should a network request fail due to a 429 or 5xx error response from the server, the SDK will (by default) automatically retry the request up to {{defaultMaxRetry}} times, waiting no less than {{defaultMinWaitInMs}} milliseconds between each attempt.
+
+You can override these default values by initializing an instance of the `RetryParams` class. It's constructor accepts two parameters: `max_retry` and `min_wait_in_ms`. The `max_retry` value assigns the maximum number of retries, and the `min_wait_in_ms` value assigns the minimum wait time between retries (in milliseconds.)
+
```

Comment:
Can you mention that the max retry value cannot exceed [{{retryMaxAllowedNumber}}](https://github.com/openfga/sdk-generator/blob/main/config/common/config.base.json#L82C4-L82C25)? (And can you please make sure we are honoring that in the SDK?)

Reply:
Good call out; I don't think the Python SDK presently honors that, but I'll double-check and cut a PR if necessary. I've added that to the template, as suggested.

Reply:
@ewanharris can we make sure to do the same for the rest of the SDKs? I believe JS, GO and .NET already honor this, I'm not sure about Java. But worth double checking.

Reply:
All readme sections now follow the new format so state this, and confirming that Java does enforce the max of 15 retries